svg/js-compiler/part_compile.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
-jsbmfbp $1
+jsbmfbp3 "$@"
```

Comment:
Should we also have an entry-point like `jsbmfbp3.bash` present to be invokable?  Do they both do the same thing?

Reply:
Not sure what you mean by this. The entry-point `jsbmfbp3.bash` is copied over and made available in `svg/js-compiler/Makefile`.